app/actions/remote/post.ts

```diff
@@ -524,13 +525,24 @@ export const fetchPostAuthors = async (serverUrl: string, posts: Post[], fetchOn
         }
     });
 
+    const isPostPriorityEnabled = await getIsPostPriorityEnabled(database);
+    const isPostAcknowledgementsEnabled = await getIsPostAcknowledgementsEnabled(database);
+    const fetchAckUsers = isPostPriorityEnabled && isPostAcknowledgementsEnabled;
+
     const usernamesToLoad = getNeededAtMentionedUsernames(existingUserNames, posts, excludeUsername);
     const userIdsToLoad = new Set<string>();
     for (const p of posts) {
         const {user_id} = p;
         if (user_id !== currentUserId) {
             userIdsToLoad.add(user_id);
         }
+        if (fetchAckUsers) {
+            p.metadata?.acknowledgements?.forEach((ack) => {
+                if (ack.user_id !== currentUserId && !existingUserIds.has(ack.user_id)) {
+                    userIdsToLoad.add(ack.user_id);
```

Comment:
@enahum you cannot have acknowledgements in a newly created post, the same as reactions.

Reply:
But we may be fetching posts other than the one of the notification, so is needed.

Reply:
@enahum then I wonder maybe it's appropriate to move it in the user list for acknowledgements, so that it will load missing profiles upon open.
What do you think? It will save duplication, and will be more failproof.

Reply:
We don't need the user profiles if we don't open the list i.e long press on the acknowledgement button.